massa-bootstrap/src/establisher.rs

```diff
@@ -77,7 +77,21 @@ pub mod types {
         /// # Argument
         /// * `addr`: `SocketAddr` we want to bind to.
         pub async fn get_listener(&mut self, addr: SocketAddr) -> io::Result<DefaultListener> {
-            Ok(DefaultListener(TcpListener::bind(addr).await?))
+            // Create a socket2 TCP listener to manually set the IPV6_V6ONLY flag
+            let socket = socket2::Socket::new(socket2::Domain::IPV6, socket2::Type::STREAM, None)?;
+
+            socket.set_only_v6(false)?;
+
+            socket.bind(&addr.into())?;
```

Comment:
Nice catch!
Just checked, the previous bind was async in case you need to make a DNS resolve to get the SocketAddr (through the [ToSocketAddrs trait](https://docs.rs/tokio/latest/tokio/net/trait.ToSocketAddrs.html). See more here: https://docs.rs/tokio/latest/src/tokio/net/tcp/listener.rs.html#101
In our case, we feed a single SocketAddr directly so we do not need to block ;)